.github/workflows/deploy-runtime-kubernetes-wrapper.yaml

```
@@ -123,52 +122,18 @@ jobs:
-          INPUTS_JSON: ${{ toJSON(github.event.inputs) || 'null' }}
-        run: |
-          result="$(node .github/scripts/construct-environments.mjs \
-            --mode studio \
-            --value "$STUDIO_ENVIRONMENTS" \
-            --inputs-json "$INPUTS_JSON" \
-            --allow-empty 'true')"
-          echo "result=$result" >> "$GITHUB_OUTPUT"
+    uses: ./.github/workflows/template-studio-construct-environments.yaml
+    with:
+      inputs: ${{ github.event_name == 'workflow_dispatch' && format('{{"environments":{0}}}', toJSON(github.event.inputs['studio-environments'])) || toJSON(github.event.inputs) }}
+      override-default-studio-environments: ${{ env.OVERRIDE_DEFAULT_STUDIO_ENVIRONMENTS }}
```

actionlint (1.7.11)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)